CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+* Pagination with composite indexes
```

Comment:
Please describe a kind of error. It wasn't working at all? Or bad result was returned?